modules/tiled-gallery/tiled-gallery.php

```diff
@@ -228,6 +228,25 @@ static function get_talaveras() {
 		return self::$talaveras;
 	}
 
+	/**
+	 * Enqueue js for our jetpack/gallery block
+	 */
+	public static function enqueue_block_editor_assets() {
+		wp_register_script(
+			'jetpack-tiled-gallery-block',
+			plugins_url( 'modules/tiled-gallery/block.js', JETPACK__PLUGIN_FILE ), // this is built as a new webpack entry point
```

Comment:
We might have to change this one to be relative to the plugins dir so it's equal to how we manage it in dotcom, see D16766-code.

Reply:
You can use `Jetpack::get_file_url_for_environment()`, which will pick between a minified and full version in Jetpack.
For example,
https://github.com/Automattic/jetpack/blob/b7c32ca7c4a9f892786ad718bc6628ef96677b98/class.jetpack.php#L975

Then, on WordPress.com, you'd add the JS file to a special mapping so that it gets enqueued from the right place. See D12054-code (scroll down to the bottom for the mapping).